### DISCUSSION TITLE
Fix: add error 451 alongside error 403 for "blocked" status to scripts

### DISCUSSION THREAD

**Discussion by Owyn** [iptv-org/community#104] [Ideas]
### Describe your issue

> 451 means that content is not available due to legal reasons. Another way of saying "geo-locked".

https://github.com/iptv-org/iptv/pull/3883#issuecomment-899639090
https://github.com/iptv-org/iptv/pull/6839#pullrequestreview-920888536

![image](https://user-images.githubusercontent.com/1309656/160094989-7954147f-e447-465b-81ca-20e90cc86b20.png)

**Comment by Owyn** [comment]
does bot currently auto-delete such geo-blocked channels after a while of them being marked as 'error' ?

but anyway, they don't even get into any public (language \ thematic) playlists atm, do them?

**Comment by freearhey** [comment]
> does bot currently auto-delete such geo-blocked channels after a while of them being marked as 'error' ?

no, bot does not delete links

> but anyway, they don't even get into any public (language \ thematic) playlists atm, do them?

yes, if a link has `status="error"` it does not get into the public playlists

**Comment by freearhey** [comment]
Done in #7476
